### ISSUE TITLE
Ruby/Sapphire ENG Pickup doesn’t work

### ISSUE THREAD

**Issue by DampieNWC** [40Cakes/pokebot-gen3#356]
Bot Mode: Pokecenter Loop
Game: Ruby
Language: English
In-Depth explanation:
- Auto Battle is on
- Pickup threshold met
- First item obtained from Zigzagoon
- The bot doesn’t press A an extra time to close the “received item from zig” message
- Stuck there until I manually close it

**Comment by pekempy** [comment]
Yeah this needs to be looked at, just raised a PR to note this on the Wiki page.
I'll keep the issue open though as a note that it needs to be looked at
